store.go

```diff
@@ -188,10 +187,7 @@ func (c *controller) watchNetworks() error {
 				lview := c.networks
 				c.Unlock()
 				for k, v := range lview {
-					global, _ := v.isGlobalScoped()
-					if global {
-						tmpview[k] = v
-					}
+					tmpview[k] = v
```

Comment:
I changed it back. I don't know how to produce a globally and locally scoped objects given each KV object is either GlobalScope or LocalScope.